client/src/pages/InsightsPage/InsightsPage.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import TaskStatusChart from "../../components/TaskStatusChart/TaskStatusChart";
+import styles from "./InsightsPage.module.css";
+import { InsightsCarousel } from "../../components/Carousel/Carousel";
+
+const InsightsPage = () => {
+  const nHoursAhead = (nHours) => {
```

Comment:
Is this function only temporarily there for the sampleTaskData? (i.e. Will this be removed once it's integrated with the rest of the insights page)

Reply:
Nevermind, I saw the comment from another page that it's a temporary function

Reply:
Yep it's a temporary function